docs/services/rais.md

```diff
@@ -0,0 +1,124 @@
+# RAIS
+
+**Contents**
+
+- [Install](#install)
+    - [Dependencies](#dependencies)
+    - [Clone from GitHub](#clone-from-github)
+    - [Configure Permissions](#configure-permissions)
+    - [Configure Docker Compose](#configure-docker-compose)
+        - [Override File](#override-file)
+    - [Add Apache Config](#add-apache-config)
+
+
+## Install
+
+### Dependencies
+
+```bash
+yum install docker docker-compose
+systemctl enable docker
+systemctl start docker
+```
+
+### Clone from GitHub
+
+```bash
+mkdir /var/local/docker
+chmod 750 /var/local/docker
+cd /var/local/docker
+git clone https://github.com/uoregon-libraries/rais-image-server rais
+
+cd rais
+
+# Switch to master branch
+git checkout master
+```
+
+### Configure Permissions
+```bash
+# Set SELinux context on files to be mounted in container
+semanage fcontext -a -t container_file_t "/var/local/docker/rais/(?:cap-max|rais-example)\.toml"
+restorecon -F -R /var/local/docker/rais
+
+semanage fcontext -a -t container_file_t "/var/local/newspapers(/.*)?"
+restorecon -F -R /var/local/newspapers
+
+# Set file permissions for newspaper files
+cd /var/local/newspapers
+chmod -R g+rwX (batch_name or * for all)
+chmod -R o+rX (batch_name or * for all)
+find . -type d -exec chmod g+s {} \;
+find . -type f -exec chmod -x {} \;
+```
+
+### Configure Docker Compose
+Define custom docker-compose config file
+
+`vim docker-compose.override.yml`:
+```yml
+version: "3.4"
+
+services:
+  rais:
+    image: uolibraries/rais
+    environment:
+      - RAIS_ADDRESS
+      - RAIS_LOGLEVEL
+      - RAIS_TILEPATH
+      - RAIS_IIIFURL
+      - RAIS_INFOCACHELEN
+      - RAIS_TILECACHELEN
+      - RAIS_IMAGEMAXAREA
+      - RAIS_IMAGEMAXWIDTH
+      - RAIS_IMAGEMAXHEIGHT
+      - RAIS_PLUGINS
+    volumes:
+      - /var/local/newspapers:/var/local/images:ro
+      - ./rais-example.toml:/etc/rais.toml:ro
+      - ./cap-max.toml:/etc/rais-capabilities.toml:ro
+    ports:
+      - 12415:12415
+    restart: always
+```
+
+Define environment file for docker-compose use which overrides defaults
+from `rais-example.toml` mounted as `/etc/rais.toml`. Default values from this
+config file are commented for reference.
+
+`vim .env`:
+```bash
+#RAIS_ADDRESS=:12415
+RAIS_LOGLEVEL=INFO
+#RAIS_TILEPATH=/var/local/images
+RAIS_IIIFURL=https://nebnewspapers(-env).unl.edu/rais
+#RAIS_INFOCACHELEN=10000
+#RAIS_TILECACHELEN=0
+#RAIS_IMAGEMAXAREA=104857600
+#RAIS_IMAGEMAXWIDTH=20480
+#RAIS_IMAGEMAXHEIGHT=20480
+# Must match at least one plugin,
+# but without other ENV var datadog plugin stays disabled
+RAIS_PLUGINS=datadog.so
+```
+
+Start RAIS docker container
+
+`docker-compose -f docker-compose.override.yml up`
```

Comment:
does this need to be backgrounded?

Reply:
It doesn't need to be, but I do prefer that. I use a bash alias, so I forgot to add it for this un-aliased version. I'll add the flag :+1: